dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/data/upload/DataUploadRunnable.kt

```diff
@@ -44,7 +44,7 @@ internal class DataUploadRunnable(
         var uploadAttempts = 0
         var lastBatchUploadStatus: UploadStatus? = null
         if (isNetworkAvailable() && isSystemReady()) {
-            val context = contextProvider.context
+            val context = contextProvider.context(withFeatureContexts = emptySet())
```

Comment:
shouldn't be emptySet() by default ?

Reply:
No, this is `ContextProvider` interface here which is internal and doesn't have default arguments (since in most of the cases it should just pass argument from the caller, like `withWriteContext`, etc.).